lib/diagnostic.js

```diff
@@ -19,15 +20,15 @@
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = true;
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = [deathStar];
```

Comment:
As you noted in the question above, a function defined in the same context as `laserQuantity` has access to that variable. So, `fire` also has access to it.
Your answer should also be in the form of strings in an array so that tests can pass.
`['deathStar', 'fire']`